components/status/StatusActions.vue

```diff
@@ -23,14 +26,36 @@ const {
   toggleReblog,
 } = $(useStatusActions(props))
 
+function recordEngagement(engagementAction: String) {
+  const analyticsId = feedName ? `${feedName}.post.${engagementAction}` : `post.${engagementAction}`
+  engagement.record({ ui_identifier: analyticsId, mastodon_status_id: status.id, ...engagementDetails[analyticsId] })
+}
+
 function reply() {
+  recordEngagement('reply')
+
   if (!checkLogin())
     return
   if (details)
     focusEditor()
   else
     navigateToStatus({ status, focusReply: true })
 }
+
+function reblog() {
+  recordEngagement('reblog')
+  toggleReblog()
+}
+
+function favourite() {
```

Comment:
using the UK English version of "favorite" to match the called `toggleFavourite`, even though there's a usage of the US English version elsewhere in this file in `useStarFavoriteIcon`

Reply:
grumble grumble, i really dislike that these have been mixed throughout the entire repo but one problem at a time i suppose